src/js/control-bar/mute-toggle.js

```diff
@@ -34,7 +34,7 @@ var MuteToggle = Button.extend({
 MuteToggle.prototype.createEl = function(){
   return Button.prototype.createEl.call(this, 'div', {
     className: 'vjs-mute-control vjs-control',
-    innerHTML: '<div><span class="vjs-control-text">' + this.localize('Mute') + '</span></div>'
+    innerHTML: `<div><span class="vjs-control-text">${this.localize('Mute')}</span></div>`
```

Comment:
See [this comment](https://github.com/videojs/video.js/pull/1976/files#r27762603)